cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -191,7 +191,7 @@
         {% elif additional_check.type == 'ownership_or_control_statement_has_is_compontent_and_component_statement_ids' %}
             <tr>
                 <td>
-                    {% trans 'An Ownership-or-control Statement cannot both be a component statement (isComponent) and have component statements (componentStatementIDs).' %}
+                    {% blocktrans %}<code>Ownership-or-control statement</code> has an <code>isComponent</code> value ('true') incompatible with having its own components in <code>componentStatementIDs</code>.{% endblocktrans %}
```

Comment:
```suggestion
                    {% blocktrans %}<code>Relationship statement</code> has an <code>isComponent</code> value ('true') incompatible with having its own components in <code>componentStatementIDs</code>.{% endblocktrans %}
```
Or has OOC been used in reference to previous versions?

Reply:
Yes - this check is for pre BODS 0.4 data.